cli/src/commands/test.ts

```diff
@@ -0,0 +1,69 @@
+import { Command, flags } from "@oclif/command";
+import { safeParse } from "../common/safe-parse";
+import { runTest, TestFilter } from "../test-utils/test-runner";
+
+const ARG_API = "spot_contract";
+
+/**
+ * oclif command to test a spot contract
+ */
+export default class Test extends Command {
+  static description = "Test a Spot contract";
+
+  static examples = ["$ spot test api.ts -u http://localhost:3000"];
+
+  static args = [
+    {
+      name: ARG_API,
+      required: true,
+      description: "path to Spot contract",
+      hidden: false
+    }
+  ];
+
+  static flags = {
+    help: flags.help({ char: "h" }),
+    url: flags.string({
+      required: true,
+      char: "u",
+      description: "Base URL"
+    }),
+    stateUrl: flags.string({
+      char: "s",
+      description: "State change URL"
```

Comment:
It'd be good to document somewhere what this endpoint should look like. I wonder if we could find a good way to describe that API endpoint... 😛

Reply:
I'll add a section about it in the Readme for now

Reply:
in a seperate PR

Reply:
Or maybe in the wiki? I think the time has come to spend effort on the documentation for other teams (and curious outsiders). We should probably have a sync about this next week.